src/app/courses/components/date-input/date-input.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="date">
+  <label class="date-lable" for="date-input">
```

Comment:
1. Typo in the word 'label'
2. According to BEM this class should be `date__label`
3. Attribute `for` is bound with the input's id attribute? not with a class.
4. The text inside the label has to be **bold**